sc-memory/sc-core/sc-store/sc_stream_memory.c

```diff
@@ -91,7 +91,6 @@ sc_result sc_stream_memory_free_handler(const sc_stream *stream)
 
   if (buffer->data_owner == SC_TRUE)
   {
-    g_assert(buffer->data != 0);
```

Comment:
Return this assert

Reply:
What's the value of this assert?

Reply:
As purpose of any other assert - check the logic. We should not go there, that data is empty.